libs/admin/src/lib/components/detail-section/detail-section.component.html

```diff
@@ -3,15 +3,9 @@
     <h2>
       {{ title }}
     </h2>
-    @if (hasButton) {
-    <button asset-sg-primary (click)="onButtonClick()">
-      {{ buttonTitle }}
-      <svg-icon [key]="buttonIcon"></svg-icon>
-    </button>
-
-    }
+    <ng-content select="[button]"></ng-content>
```

Comment:
I would call this select `prefix` or `actions`, as it doesn't necessarily contain a button.
```suggestion
    <ng-content select="[prefix]"></ng-content>
```